Formula/ipopt.rb

```diff
@@ -0,0 +1,50 @@
+class Ipopt < Formula
+  desc "Interior point optimizer"
+  homepage "https://projects.coin-or.org/Ipopt/"
+  url "https://www.coin-or.org/download/source/Ipopt/Ipopt-3.12.12.tgz"
+  sha256 "7baeb713ef8d1999bed397b938e9654b38ad536406634384455372dd7e4ed61f"
+  head "https://github.com/coin-or/Ipopt.git"
+
+  depends_on "gcc"
+
+  def install
+    ENV.delete("MPICC")
+    ENV.delete("MPICXX")
+    ENV.delete("MPIFC")
+    ENV.deparallelize
```

Comment:
Why is parallel build not supported? Is this a known bug?

Reply:
Technically, parallel install is not supported. I can probably get the build to work in parallel.